zap/src/main/java/org/zaproxy/zap/extension/pscan/PassiveScanThread.java

```diff
@@ -213,12 +213,10 @@ public void run() {
                                         Stats.incCounter("stats.pscan.reqBodyTooBig");
                                         if (logger.isDebugEnabled()) {
                                             logger.debug(
-                                                    "Request to "
-                                                            + msg.getRequestHeader().getURI()
-                                                            + " body size "
-                                                            + msg.getRequestBody().length()
-                                                            + " larger than max configured "
-                                                            + maxBodySize);
+                                                    "Request to {}  body size {} larger than max configured {}",
```

Comment:
2 spaces here - not a deal breaker, but thought I'd mention it anyway ;)

Reply:
Will fix